bootstrap.sh

```diff
@@ -32,6 +32,8 @@ if [ -n "$CMD" ]; then
     git clean -ffdx
 
     exit 0
+  elif [ "$CMD" = "full" ]; then
+    export NO_CACHE=1
```

Comment:
This seems to be only supported at top level. For subprojects you have to use NO_CACHE=1 to prevent use of cache. Would it make sense to have one or the other for all files?